### PR TITLE
fix: shared post card metadata

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes

@rebelchris the build was failing due to some existing implementations that require updating.
Though instead of refactoring the whole component, I have instead wrapped the only usage that required the clamping. Would love to hear what you think of the changes.

Local preview:
image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1107 #done

### CODE REVIEW

**Comment by sshanzel** [comment]
We have a test suite for this component and a case specific to the issue we suffered here, I wonder if there is any way we can replicate what we had here in a test environment. Has anyone tried it before? Like, be able to set some specific width on test environment for testing a component?

**Review by rebelchris — CHANGES_REQUESTED** [review]
This actually doesn't clamp it.

Screenshot 2023-02-24 at 09 49 30

**Comment by sshanzel** [comment]
@rebelchris oh, I didn't see that coming. Let me revert the revert quickly! And just fix the build 😅

**Comment by sshanzel** [comment]
@rebelchris did it work for you locally? It doesn't seem to fix the initial concern on my local environment.

image

**Comment by sshanzel** [comment]
@rebelchris I can continue to debug this forward if you won't have time for it today. Just let me know 🙌

**Comment by sshanzel** [comment]
@rebelchris I've realized the problem is with the separator taking a new line when rendered. So I thought of using the browser-supported text that is intended for separations like this. It is called `interpunct`. I think this is best to use than creating a div element just to replicate the behavior. The card should now look like below on normal content:

image

Looks like this when clamped. Since the clamp is for a single line only, we need to trigger the breaking of words:
image

**Review by rebelchris — APPROVED** [review]
@sshanzel Just one question, but nice find!
That's a good solution for this edge case.

File: packages/shared/src/components/ConditionalWrapper.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ReactElement } from 'react';
 
-interface ConditionalWrapperProps {
+export interface ConditionalWrapperProps {
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/shared/src/components/ConditionalWrapper.tsx:3]: Oh, this was on some previous implementation. I can revert this back now. Thanks for the note!

**Pull request merged**